pkg/testutils/moduletemplate.go

```diff
@@ -32,7 +31,7 @@ func GetModuleTemplate(ctx context.Context,
 	clnt client.Client,
 	module v1beta2.Module,
 	kyma *v1beta2.Kyma,
-) (*v1beta2.ModuleTemplate, error) {
+) (*templatelookup.ModuleTemplateInfo, error) {
```

Comment:
Please don't change this return type, if return `emplatelookup.ModuleTemplateInfo`, you can't use it for ModuleTemplate update. It will cause error in `SetModuleTemplateBetaLabel`, this function is expected to return a real `ModuleTemplate`, then do `clnt.Update(ctx, moduleTemplate)`

```
SetModuleTemplateBetaLabel attempt failed: failed to update module template: no kind is registered for the type templatelookup.ModuleTemplateInfo in scheme "pkg/runtime/scheme.go:110" 
```

This function is `GetModuleTemplate`, the expected returned type is `ModuleTemplate`, create another function if you want a `ModuleTemplateInfo`